src/app/@ansyn/ansyn/modules/menu-items/tools/export-maps-popup/export-maps-popup.component.html

```diff
@@ -1,24 +1,70 @@
-<div class="dialog-area">
-	<div mat-dialog-content>
-		<div mat-dialog-title>
-			<button class="close" mat-button (click)="closeModal()">X</button>
-			<h1 mat-dialog-title class="title-message">{{title | translate}} {{credentialsService?.user}}</h1>
-		</div>
-		<mat-progress-bar *ngIf="!isDownloadAvailable" class="main-progress" color="primary" mode="indeterminate"></mat-progress-bar>
-		<br>
-		<p mat-dialog-title class="info">
-			{{description | translate}}
-		</p>
-		<div class="btn-area">
-			<div class="download-btn">
-				<button mat-button (click)="onClick.emit(true)" [disabled]="!isDownloadAvailable">
-					<i class="fa fa-cloud-download" aria-hidden="true"></i>
-					{{'Download' | translate}}</button>
-			</div>
-			<div class="cancel-btn">
-				<button mat-button (click)="onClick.emit(false)">{{'Cancel' | translate}}</button>
+<div class="header">
+	<h1 mat-dialog-title>{{title | translate}}</h1>
+	<button mat-dialog-close tabindex="-1" (click)="close()">X</button>
+</div>
+<mat-dialog-content>
+	<h5 mat-dialog-title>{{description}}</h5>
+	<div class="method-chose">
+		<ansyn-radio
+			[(ngModel)]="exportMethod"
+			[value]="basicExport"
+			name="basicExport">
+			{{basicExport | translate}}
+		</ansyn-radio>
+		<ansyn-radio
```

Comment:
maybe create a config for the basic and advanced exports, and use ngFor

Reply:
👍